source/administration/console/managing-objects.rst

```diff
@@ -84,7 +83,7 @@ While creating a bucket, you can enable :ref:`versioning <minio-bucket-versionin
 
    If you enable versioning, you can specify prefixes to exclude from versioning.
 
-You **must** configure replication, locking, and versioning options at the time of bucket creation.
+You **must** configure locking and versioning options at the time of bucket creation.
```

Comment:
In context this works OK, but it's not as clear as it could be (I tripped over it until I expanded upward) ... if you feel like it, try something like

```suggestion
You can configure locking and versioning options only when you create the bucket.
```